firebase-crashlytics/CHANGELOG.md

```diff
@@ -1,4 +1,7 @@
 # Unreleased
+* [fixed] Fixed NPE if firebase-common is at too low a version
```

Comment:
May a suggest rewording this as:
```suggestion
* [fixed] Fixed NPE caused by an older version of `firebase-common`
```

or even be more explicit and say:
```suggestion
* [changed] Updated to depend on `firebase-common:20.3.0`.
```

Reply:
updated to match rachel's spec